### ISSUE TITLE
[Playbook Repositories] fatal: could not read Username for xxx: No such device or address

### ISSUE THREAD

**Issue by Commifreak** [ansible-semaphore/semaphore#835]
Hi,

I set up a new template with Playbook repo. After execute the template I get:

```
 10:04:05 AMTask 2147483642 added to queue
10:04:05 AMPreparing: 2147483642
10:04:05 AMPrepare task with template: test
10:04:05 AMCloning repository https://mygit/myrepo.git
10:04:05 AMCloning into 'repository_1_1'...
10:04:05 AMfatal: could not read Username for 'https://mygit/myrepo.git': No such device or address
10:04:05 AMFailed updating repository: exit status 128
```

I dont see whats wrong.

The repo and a key (user password) is configured and set.

![grafik](https://user-images.githubusercontent.com/87267/151326986-ab85b9b2-9327-42c0-a595-406ebcc2e879.png)

**Comment by fiftin** [comment]
Hi @Commifreak ,

What version do you use (number and how installed)?

**Comment by fiftin** [comment]
I will try to reproduce.

**Comment by Commifreak** [comment]
> Hi @Commifreak ,
> 
> What version do you use (number and how installed)?

2.8.34 via snap

**Comment by fiftin** [comment]
Fixed in v2.8.35, currently in beta on Snapcraft, you can install it by command:
```
sudo snap install semaphore --beta
```
Release details: https://github.com/ansible-semaphore/semaphore/releases/tag/v2.8.35
